### DISCUSSION TITLE
本地修改器迁移

### DISCUSSION THREAD

**Discussion by MCLONG0** [dyang886/Game-Cheats-Manager#12] [Ideas]
大佬牛逼，我有个想法，能不能做个一键搜索本地修改器，然后把修改器迁移到管理器文件夹内的功能

**Comment by Karasukaigan** [comment]
我查看了代码，列出修改器的方法就是直接读取GCM Trainers目录里的所有exe文件，所以你只需要将自己下载的修改器复制进目录里就行，甚至可以是非风灵月影的修改器！

**Comment by Karasukaigan** [comment]
我在菜单里添加了一个导入选项，可以实现从本地导入已有的修改器，刚提交了合并请求，还得等 @dyang886 来处理。

**Comment by Frostpung** [comment]
如果要下载多个修改器的话，将信息输出的表和下载列表分离应该会好点，增加一个语料库，例如钢铁雄心，需要填入Heart of iron 4才能搜索到，如果使用语料库应该可以提高搜索准度

**Reply by Frostpung** [reply]
仅是一个建议，没有别的意思
